deeptime/markov/msm/tram/_bindings/src/tram_module.cpp

```diff
@@ -22,16 +22,14 @@ PYBIND11_MODULE(_tram_bindings, m) {
                 .def_property_readonly("modified_state_counts_log", &TRAM<double>::modifiedStateCountsLog)
                 .def_property_readonly("lagrangian_mult_log", &TRAM<double>::lagrangianMultLog)
                 .def_property_readonly("therm_state_energies", &TRAM<double>::thermStateEnergies)
-                .def_property_readonly("markov_state_energies", &TRAM<double>::markovStateEnergies)
-                .def("compute_log_likelihood", &TRAM<double>::computeLogLikelihood, py::call_guard<py::gil_scoped_release>());
-
+                .def_property_readonly("markov_state_energies", &TRAM<double>::markovStateEnergies);
 
         py::class_<TRAMInput<double>, std::shared_ptr<TRAMInput<double>>>(tramMod, "TRAMInput").def(
-                py::init<deeptime::np_array_nfc<int> &&, deeptime::np_array_nfc<int> &&, DTrajs, BiasMatrices<double>>(),
-                "state_counts"_a, "transition_counts"_a, "dtrajs"_a, "bias_matrices"_a);
+                py::init<deeptime::np_array_nfc<int> &&, deeptime::np_array_nfc<int> &&, DTraj, BiasMatrix<double>>(),
+                "state_counts"_a, "transition_counts"_a, "dtraj"_a, "bias_matrix"_a);
 
-        tramMod.def("compute_sample_weights", &computeSampleWeights<double>, py::call_guard<py::gil_scoped_release>(),
-                    "therm_state_index"_a = -1, "dtrajs"_a, "bias_matrices"_a, "therm_state_energies"_a,
+        tramMod.def("compute_sample_weights", &computeSampleWeightsLog<double>, py::call_guard<py::gil_scoped_release>(),
```

Comment:
the bound method is called `computeSampleWeightsLog`, the exported method name is `compute_sample_weights` - which one is it? Logspace or linspace?